homework1/homework1.css

```diff
@@ -0,0 +1,17 @@
+* {margin: 0; padding: 0;}
+body {
+    background-color: blue;
+}
+
+.one {
+    text-align: center;
+    align-items: center;
```

Comment:
<img width="920" alt="Screen Shot 2022-01-22 at 20 42 52" src="https://user-images.githubusercontent.com/1973623/150640802-a54b8d12-7d15-4468-8cb7-5265a231c033.png">

It should be like this because I am requesting it must be centralized on the page. (Nó nên như thế này này, anh yêu cầu là nó ở giữa màn hình, chứ ko phải ở trên top và ở giữa)

Should update like this and it will work. (em update code lại như thế này là ổn - Keyword cần học - Flexbox css)
```css
body {
  background-color: blue;
  height: 100vh;
}

.one {
    justify-content: center;
    align-items: center;
    display: flex;
    height: 100%;
}
```

Reply:
i will fix it and learn flexbox css